SystemInterface/Diagnostics/IProcess.cs

```diff
@@ -73,6 +78,27 @@ public interface IProcess
         [MonitoringDescription("ProcessStandardOutput"), DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden), Browsable(false)]
         IStreamReader StandardOutput { get; }
 
+        /// <summary>
+        ///  Gets a new System.Diagnostics.Process component and associates it with the currently active process.
+        /// </summary>
+        /// <returns>A new System.Diagnostics.Process component associated with the process resource that is running the calling application.</returns>
+        Process GetCurrentProcess();
```

Comment:
Implementation of this method calls static Process.GetCurrentProcess which returns object of Process class. If we would like to return IProcess we would have to return here ProcessWrap instance or any other class implementing IProcess that would have to be created.
This would make the code a little bit more complicated. Although there are benefits for returning IProcess here, I believe that in majority of cases current implementation should be enough.

If someone would see benefits for himself having IProcess being returned I'm happy to implement it. For our needs it's enough to just return Process object